server/base/src/main/java/org/apache/accumulo/server/master/recovery/RecoveryPath.java

```diff
@@ -18,13 +18,19 @@
  */
 package org.apache.accumulo.server.master.recovery;
 
+import org.apache.accumulo.server.ServerContext;
+import org.apache.accumulo.server.cli.ServerUtilOpts;
 import org.apache.accumulo.server.fs.VolumeManager.FileType;
 import org.apache.hadoop.fs.Path;
 
 public class RecoveryPath {
 
   // given a wal path, transform it to a recovery path
   public static Path getRecoveryPath(Path walPath) {
+
+    ServerUtilOpts opts = new ServerUtilOpts();
+    ServerContext context = opts.getServerContext();
```

Comment:
This isn't the right way to get the ServerContext. The caller should pass in the context reference that they have, as an additional parameter to this method. I see two callers for this method. `TabletServer` should have a `getContext()` method on it, and `RecoveryManager` can use `master.getContext()`.